torch/testing/_internal/distributed/distributed_test.py

```diff
@@ -7112,14 +7111,8 @@ def test_ddp_inference(self):
             for test in tests:
                 test_model, test_local_model, test_inp = test
                 if self.rank == 0:
-                    with torch.no_grad():
```

Comment:
Does this mean `no_grad` is no longer relevant for inference and `.eval()` will sufficient for all cases?